blob/blob.go

```diff
@@ -498,11 +498,17 @@ func (b *Bucket) NewRangeReader(ctx context.Context, key string, offset, length
 		opts = &ReaderOptions{}
 	}
 	dopts := &driver.ReaderOptions{}
+	tctx := trace.StartSpan(ctx, "gocloud.dev/blob.NewRangeReader")
+	defer func() {
+		if err != nil {
+			trace.EndSpan(tctx, err)
+		}
+	}()
 	r, err := b.b.NewRangeReader(ctx, key, offset, length, dopts)
 	if err != nil {
-		return nil, wrapError(b.b, err)
+		err = wrapError(b.b, err)
```

Comment:
Is this changed needed? Isn't err always set to the return value?
https://play.golang.org/p/SbrHLsHoGoD